src/scss/global.scss

```diff
@@ -0,0 +1,9 @@
+:root {
+  @extend %component-timing;
```

Comment:
Adding CSS comments here will be a nice QoL for inspecting these vars in the debug tools in browser. Will help provide context.